csbuild/tools/project_generators/visual_studio/__init__.py

```diff
@@ -131,6 +133,11 @@ def defines(self):
 		"""Project defines"""
 		return self._defines
 
+	@property
+	def ccLanguageStandard(self):
```

Comment:
How does this difference from `cxxLanguageStandard` exactly? They both have the exact same comment.